### DISCUSSION TITLE
Not scanning appsettings files?

### DISCUSSION THREAD

**Discussion by JGSilverman** [zricethezav/gitleaks#957] [Q&A]
I'm using the Azure DevOps build task for this (version 2), and while it is picking up keys/tokens in files, it's not picking up a key (which I know because I can see it), in an appSettings.json file inside one of the projects in the solution.

I don't see much documentation on the logging. How can I see a list of every file the task scanned to ensure it's even scanning the appSettings files? (verbose = true and logoptions='debug') doesn't achieve that.

**Comment by Simon-Wyatt** [comment]
I'm seeing the same issue. Using a relatively simple regex rule, looking for "pass" followed by 0-5 letters, I cannot detect a plaintext secret of the form:
    _pass password_
in my pipeline.

However, if I pull the repo to my local machine, I can detect it using the same configuration rule: 
    _(?i)pass*[a-z]{0,5}[:|=|\s]_
